setup.py

```diff
@@ -1,94 +1,4 @@
-from os import path
-from setuptools import setup, find_packages
+from setuptools import setup
```

Comment:
I can also note that this `setup.py` is probably why the CI is failing because it expects `setuptools` but that is not required for the build.

`pip` documentation mentions that the following is the default build system:
```
[build-system]
requires = ["setuptools>=40.8.0", "wheel"]
build-backend = "setuptools.build_meta:__legacy__"
```